src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1012,6 +1012,20 @@ public static void onEmptyLeftClick(EntityPlayer player, ItemStack stack)
         MinecraftForge.EVENT_BUS.post(new PlayerInteractEvent.LeftClickEmpty(player, stack));
     }
 
+    public static net.minecraftforge.event.entity.player.CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical)
+    {
+        net.minecraftforge.event.entity.player.CriticalHitEvent hitResult = new net.minecraftforge.event.entity.player.CriticalHitEvent(player, target);
```

Comment:
No i thing this is not so good. Because if I a make an alwaiys-critical-sword it would be annyoing figuring out I have to set the default multiplier by my self.